thirdparty/haw_logger/logger/log_base.h

```diff
@@ -42,7 +42,7 @@
 #define DEPRECATED LOGD() << "This function deprecated!!"
 #define DEPRECATED_USE(use) LOGD() << "This function deprecated!! Use:" << use
```

Comment:
same

Reply:
I think I'll rather remove them altogether. They are not used, as far as I can find.

Reply:
They are taken from the original source. Maybe getting them fixed there is in order, @igorkorsukov ?

Reply:
Thanks.
I will fix them later